db/migrate/20170307190555_add_min_events_option_to_peak_detector_agents.rb

```diff
@@ -0,0 +1,19 @@
+class AddMinEventsOptionToPeakDetectorAgents < ActiveRecord::Migration[5.0]
+  def up
+    Agents::PeakDetectorAgent.find_each do |agent|
+      if agent.options['min_events'].nil?
+        agent.options['min_events'] = '4'
+        agent.save!
```

Comment:
We should do `save(validate: false)` so that the migration can't error if for some reason their agent is invalid.  Same in the `down`.